scripts/dts/edtlib.py

```diff
@@ -562,14 +569,9 @@ def _init_prop(self, name, options):
         if prop.description:
             prop.description = prop.description.rstrip()
         prop.val = val
-        enum = options.get("enum")
         if enum is None:
             prop.enum_index = None
         else:
-            if val not in enum:
-                _err("value ({}) for property ({}) is not in enumerated "
-                     "list {} for node {}".format(val, name, enum, self.name))
-
             prop.enum_index = enum.index(val)
```

Comment:
Can shorten a bit with

```python
prop.enum_index = None if enum is None else enum.index(val)
```

Reply:
The `None` is to distinguish from an empty list by the way (though it's kinda pointless to have an empty list for `enum`).

Another option would be to require `enum` to have at least one element (can check that with `if enum`, as empty lists are falsy).

Reply:
Done.